chirp/drivers/baofeng_common.py

```diff
@@ -448,8 +448,8 @@ def bcd_decode_freq(bytes):
     real_freq = 0
     for byte in bytes:
         if byte > 9:
-            msg = ("Found Invalid BCD Encoding")
-            raise InternalError(msg)
+            LOG.debug('Found Invalid BCD Encoding')
+            return chirp_common.format_freq(0)
```

Comment:
I think this is the wrong thing to do here. I think this generalized helper *should* fail when we can't decode something that should be BCD. Instead, we should catch the failure where it happens and make the decision there about what to do. I think the way to tell it's wrong is that we'll see a random "Invalid BCD encoding" in the log, but we won't know what was actually wrong. Would doing it differently be okay?

So if you don't change this and leave the raise, put the exception handling where you use it:
```
try:
    value = bfc.bcd_decode_freq(foo)
except settings.InternalError:
    LOG.debug('Failed to decode VFOb')  # or whatever
    value = chirp_common.format_freq(0)
```
What do you think?

Reply:
Let me give it a try. Thanks. It might not be until tomorrow evening.

Reply:
My initial testing looks like this is going to work. I will expand it to the other 5 subVFO settings and perform better testing tomorrow. Thanks again!

Reply:
Cool, thanks!